src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/InitialConsiderationController.java

```diff
@@ -54,4 +58,29 @@ public ResponseEntity<CCDCallbackResponse> completeInitialConsideration(@Request
                 COMPLETE_IC_BODY).
             build());
     }
+
+    @PostMapping(value = "/startInitialConsideration", consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "start the Initial Consideration flow")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "Accessed successfully", content = {
+            @Content(mediaType = "application/json", schema = @Schema(implementation = CCDCallbackResponse.class))}),
+            @ApiResponse(responseCode = "400", description = "Bad Request"),
+            @ApiResponse(responseCode = "500", description = "Internal Server Error")})
+    public ResponseEntity<CCDCallbackResponse> startInitialConsideration(@RequestBody CCDRequest ccdRequest,
+                                                                         @RequestHeader(value = "Authorization")
+                                                                                 String userToken) {
+        log.info("START OF INITIAL CONSIDERATION FOR CASE ---> " + ccdRequest.getCaseDetails().getCaseId());
```

Comment:
Logs these days are better prepared with parameters rather than appending strings together.  I.e.
log.info("START OF INITIAL CONSIDERATION FOR CASE ---> ", ccdRequest.getCaseDetails().getCaseId());